2021-05-09 00:39:13,288 INFO    MainThread:20900 [wandb_setup.py:_flush():69] setting env: {}
2021-05-09 00:39:13,288 INFO    MainThread:20900 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 00:39:13,288 INFO    MainThread:20900 [wandb_init.py:_log_setup():336] Logging user logs to /home/trenton-windows/repos/spicy-dann/wandb/run-20210509_003913-3824jtvb/logs/debug.log
2021-05-09 00:39:13,288 INFO    MainThread:20900 [wandb_init.py:_log_setup():337] Logging internal logs to /home/trenton-windows/repos/spicy-dann/wandb/run-20210509_003913-3824jtvb/logs/debug-internal.log
2021-05-09 00:39:13,288 INFO    MainThread:20900 [wandb_init.py:init():369] calling init triggers
2021-05-09 00:39:13,288 INFO    MainThread:20900 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-09 00:39:13,289 INFO    MainThread:20900 [wandb_init.py:init():418] starting backend
2021-05-09 00:39:13,290 INFO    MainThread:20900 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-09 00:39:13,303 INFO    MainThread:20900 [backend.py:ensure_launched():132] starting backend process...
2021-05-09 00:39:13,310 INFO    MainThread:20900 [backend.py:ensure_launched():137] started backend process with pid: 20928
2021-05-09 00:39:13,314 INFO    MainThread:20900 [wandb_init.py:init():423] backend started and connected
2021-05-09 00:39:13,316 INFO    MainThread:20900 [wandb_init.py:init():465] updated telemetry
2021-05-09 00:39:13,317 INFO    MainThread:20900 [wandb_init.py:init():484] communicating current version
2021-05-09 00:39:13,999 INFO    MainThread:20900 [wandb_init.py:init():489] got version response 
2021-05-09 00:39:13,999 INFO    MainThread:20900 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-09 00:39:14,152 INFO    MainThread:20900 [wandb_init.py:init():522] starting run threads in backend
2021-05-09 00:39:16,536 INFO    MainThread:20900 [wandb_run.py:_console_start():1565] atexit reg
2021-05-09 00:39:16,537 INFO    MainThread:20900 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-09 00:39:16,537 INFO    MainThread:20900 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-09 00:39:16,540 INFO    MainThread:20900 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-09 00:39:16,540 INFO    MainThread:20900 [wandb_init.py:init():546] run started, returning control to user process
2021-05-09 00:39:16,542 INFO    MainThread:20900 [wandb_run.py:_config_callback():787] config_cb None None {'dataset': 'camelyon17', 'data_root': './WILDS', 'model_name': 'resnet18', 'batch_size': 16, 'n_epochs': 1, 'get_train_metrics': True, 'save_every': 5, 'run_name': 'dev-small', 'max_val_batches': 5}
2021-05-09 00:42:26,517 INFO    MainThread:20900 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-09 00:42:26,520 INFO    MainThread:20900 [wandb_run.py:_restore():1507] restore
2021-05-09 00:42:28,530 INFO    MainThread:20900 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 2
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 36779
  total_bytes: 36779
}

2021-05-09 00:42:28,751 INFO    MainThread:20900 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 2
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 36779
  total_bytes: 36779
}

2021-05-09 00:42:29,257 INFO    MainThread:20900 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 36779
  total_bytes: 40630
}

2021-05-09 00:42:29,433 INFO    MainThread:20900 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 36781
  total_bytes: 40630
}

2021-05-09 00:42:29,536 INFO    MainThread:20900 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 40630
  total_bytes: 40630
}

2021-05-09 00:42:29,638 INFO    MainThread:20900 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 40630
  total_bytes: 40630
}

2021-05-09 00:42:29,741 INFO    MainThread:20900 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 40630
  total_bytes: 40630
}

